tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -0,0 +1,289 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with Workbenches
+Resource       ../../../../Page/Components/Components.resource
+Resource       ../../../../Common.robot
+Resource       Projects.resource
+
+
+*** Variables ***
+${WORKBENCH_SECTION_XP}=             xpath=//div[div//h4[@id="workbenches"]]
+${WORKBENCH_CREATE_BTN_XP}=           xpath=//button[text()="Create workbench"]
+${WORKBENCH_CREATE_BTN_2_XP}=         xpath=//button[@id="create-button"]
+${WORKBENCH_NAME_INPUT_XP}=               xpath=//input[@name="workbench-name"]
+${WORKBENCH_DESCR_TXT_XP}=                xpath=//textarea[@name="workbench-description"]
+${WORKBENCH_IMAGE_MENU_BTN_XP}=           xpath=//section[@id="notebook-image"]//button[@aria-label="Options menu"]
+${WORKBENCH_IMAGE_ITEM_BTN_XP}=           xpath=//ul[@id="workbench-image-stream-selection"]/li/button
+${WORKBENCH_SIZE_MENU_BTN_XP}=           xpath=//section[@id="deployment-size"]//button[@aria-label="Options menu"]
+${WORKBENCH_SIZE_ITEM_BTN_XP}=           xpath=//ul[@id="container-size"]/li/button
+${WORKBENCH_ADD_VAR_BTN_XP}=           xpath=//button[text()="Add variable"]
+${WORKBENCH_STATUS_STOPPED}=                  Stopped
+${WORKBENCH_STATUS_RUNNING}=                  Running
+${WORKBENCH_STATUS_STARTING}=                 Starting...
+${WORKBENCH_STOP_BTN_XP}=                 xpath=//button[text()="Stop workbench"]
+&{IMAGE_ID_MAPPING}=                 Minimal Python=s2i-minimal-notebook    CUDA=minimal-gpu   PyTorch=pytorch
+...                                  Standard Data Science=s2i-generic-data-science-notebook    TensorFlow=tensorflow
+${KEYVALUE_TYPE}=        Key / value
+${ENV_VARIABLES_SECTION_XP}=        xpath=//section[@aria-label="Environment variables"]
+
+
+*** Keywords ***
+Create Workbench
+    [Documentation]     Creates a new workbench in a Data Science project. It assumes
+    ...                 the DS Project deta. It allows to add new or existent PV storage,
```

Comment:
_It assumes the DS Project deta._ Do you mean data?

Reply:
yes, thx